src/components/forms/AutocompleteField/AutocompleteField.js

```diff
@@ -8,6 +8,17 @@ export function FormikAutocompleteTextField({ children, options, ...props }) {
   const { onChange, onBlur, value, disabled, name, ...rest } = fieldToTextField(props);
   const { setFieldValue } = useFormikContext();
 
+  // The Autocomplete component appears to override the "autocomplete" attribute, setting it to "off"
+  // no matter how you set the autoComplete property in the component. Chrome, however, ignores
+  // autocomplete: "off" and only really turns off autofill if you set it to autocomplete: "new-password".
+  // This code uses an effect to directly change the autocomplete attribute after the component has
+  // updated.
+  const myRef = React.createRef();
+  useEffect(() => {
+    const input = myRef.current && myRef.current.getElementsByTagName('input')[0];
+    input.autocomplete = 'new-password';
+  });
```

Comment:
It would be worth adding dependency array here so that it only runs on mount and not every re-render.  Tested it locally and didn't see any issues (not that there still couldn't be 🙂 )

Docs for this are here: https://reactjs.org/docs/hooks-effect.html#tip-optimizing-performance-by-skipping-effects. 
```suggestion
  }, []);
```

Reply:
Confirmed and fixed.